src/main/java/tree/ImpTargetTreeNodeComparator.kt

```diff
@@ -0,0 +1,9 @@
+package tree
+
+class ImpTargetTreeNodeComparator : Comparator<ImpTargetTreeNode> {
+    override fun compare(o1: ImpTargetTreeNode?, o2: ImpTargetTreeNode?): Int {
+        if (o1 == null || o2 == null)
+            return 0
+        return o1!!.getId().compareTo(o2!!.getId())
```

Comment:
This is not necessary.
```suggestion
        return o1.getId().compareTo(o2.getId())
```

Reply:
Yes, you are right.

Reply:
![image](https://user-images.githubusercontent.com/15882623/76347318-55dd4080-62fe-11ea-8ce7-5e3272789ec8.png)